### PR TITLE
Feature 11 create edit post

### PR DESCRIPTION
**Pull request by yodeco**
1. WYSWYG editor
2. Save post to DB
3. Edit post
4. Preview post
5. View posts

### CODE REVIEW

**Comment by YuliaFursovych** [comment]
When you unchecked "Pinned" when editing, it is not removed from the "Posts".

**Comment by YuliaFursovych** [comment]
When you press "Save Post" button, you will be logged out.  But the post will be created/edited.

File: client/src/router.js [hunk 1]
```diff
@@ -32,6 +33,8 @@ const router = ({ childProps }) => {
             <AppliedRoute path="/room/:id/add-event" component={AddEvent} props={childProps} />
             <AppliedRoute path="/room/:id" exact component={RoomPage} props={childProps} />
             <AppliedRoute path="/room/:id/edit" exact component={EditRoom} props={childProps} />
+            <AppliedRoute path="/room/:id/edit" exact component={EditRoom} props={childProps} />
```
Comment (IgorPopach) [inline client/src/router.js]: same code: line 35 &36

**Review by IgorPopach — APPROVED** [review]

File: client/src/components/EditProfile/EditProfile.js [hunk 2]
```diff
@@ -303,6 +303,7 @@ class EditProfile extends Component {
             })
     }
     sendUserData = (data, callback) => {
+        debugger;
```
Comment (Blager89) [inline client/src/components/EditProfile/EditProfile.js:306]: I think it needs to be deleted

File: client/src/components/RoomPage/RoomPage.js [hunk 3]
```diff
@@ -51,6 +54,8 @@ class RoomPage extends React.Component {
 
     componentDidMount() {
         const { id } = this.props.match.params;
+
+        // Load MySQL DB room data;
         axios.get("/api/room/" + id)
             .then(res => {
                 console.log('response', res.data);
```
Comment (Blager89) [inline client/src/components/RoomPage/RoomPage.js:61]: delete this console

**Review by YuliaFursovych — APPROVED** [review]

File: client/src/components/RoomPage/RoomPage.js [hunk 4]
```diff
@@ -138,6 +153,33 @@ class RoomPage extends React.Component {
            console.log(err);
```
Comment (Blager89) [inline client/src/components/RoomPage/RoomPage.js:153]: again

File: client/src/components/RoomPage/RoomPage.js [hunk 5]
```diff
@@ -125,7 +140,7 @@ class RoomPage extends React.Component {
           .catch(err => {
            console.log(err);
```
Comment (Blager89) [inline client/src/components/RoomPage/RoomPage.js:141]: again

File: client/src/components/WYSWYGeditor/WYSWYGeditor.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+const uploadUrl = 'https://40132.cke-cs.com/easyimage/upload/';
+const tokenUrl = 'https://40132.cke-cs.com/token/dev/51bEORgZKne2yb0yBRm8XtbuSePrpKBbHLPkcsKcA2z3pUYhCzks1PXkwvRg';
+class WYSWYGeditor extends Component {
+    state = {
+        editorData: this.props.editorSettings.data
+        ? this.props.editorSettings.data
+        : ''
+    }
+    onChange = (event, editor) => {
+      console.log(1111);
```
Comment (Blager89) [inline client/src/components/WYSWYGeditor/WYSWYGeditor.js:16]: again

**Review by Blager89 — APPROVED** [review]

**Pull request merged**